diff --git a/imglib.h b/imglib.h
index 2a03f93..e5468c4 100644
--- a/imglib.h
+++ b/imglib.h
@@ -98,7 +98,7 @@ union image_id_index {
 	bool operator==(const image_id_index& other) const { return id == other.id; }
 	bool operator!=(const image_id_index& other) const { return id != other.id; }
 
-	operator size_t&() { return index; }
+	//operator size_t&() { return index; }
 	operator imageId&() { return id; }
 };
 
